IntegrationTests/Braintree-API-Integration-Specs/BTGraphQLHTTP_SSLPinning_IntegrationTests.swift

```diff
@@ -5,11 +5,18 @@ import XCTest
 class BTGraphQLHTTP_SSLPinning_IntegrationTests : XCTestCase {
 
     func testBTGraphQLHTTP_whenUsingProductionEnvironmentWithTrustedSSLCertificates_allowsNetworkCommunication_toBraintreeAPI() {
-        let graphqlHttp = BTGraphQLHTTP(url: URL(string: "https://payments.braintree-api.com")!, tokenizationKey: "")
+        let json = BTJSON(value: [
+            "clientApiUrl": "https://fake-client-api-url.com/path",
```

Comment:
It doesn't matter if the clientApiURL is included, since we initialize a `BTGraphQLHTTP` it will only use the graphQL URL.

However, I pushed a commit to use the same syntax as the BTHTTP SSL tests, which is a few less lines of code: e3c78fcef55d1e33cbdbfa2e26c1e6a4ba18998c